api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentCalculator.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.dhsc.htbhf.claimant.service.payments;
+
+import org.springframework.beans.factory.annotation.Value;
+import uk.gov.dhsc.htbhf.claimant.entitlement.PaymentCycleVoucherEntitlement;
+
+public class PaymentCalculator {
+
+    private final int numberOfCalculationPeriods;
+    private final int calculationPeriodThreshold;
+
+    public PaymentCalculator(@Value("${payment-cycle.number-of-calculation-periods}") int numberOfCalculationPeriods) {
+        this.numberOfCalculationPeriods = numberOfCalculationPeriods;
+        this.calculationPeriodThreshold = numberOfCalculationPeriods * 2;
```

Comment:
This needs to be configurable. I think we probably want this configured as the number of calculation periods that make up the threshold, rather than the multiplier. Naming, as ever, might be tricky. `payment-cycle.max-balance-calculation-periods`, perhaps? It'll need some description (ideally in application.yml as well as here).

Reply:
With help from Kate we came up with `number-of-calculation-periods`